test/e2e/serial/tests/configuration.go

```diff
@@ -207,11 +207,12 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 				Expect(err).ToNot(HaveOccurred())
 
 				By("waiting for mcp to start updating")
-				waitForMcpsCondition(fxt.Client, context.TODO(), mcps, machineconfigv1.MachineConfigPoolUpdating)
+				err = waitForMcpsCondition(fxt.Client, context.TODO(), mcps, machineconfigv1.MachineConfigPoolUpdating)
```

Comment:
nit:
More elegant way for handling such functions is:
```
Expect(waitForMcpsCondition(fxt.Client, context.TODO(), mcps, machineconfigv1.MachineConfigPoolUpdating)).To(Succeed())
```

Reply:
right, I'll adjust the instances, thanks!